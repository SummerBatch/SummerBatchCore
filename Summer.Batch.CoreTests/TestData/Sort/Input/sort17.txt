Person145
Person259
Person368
Person437
Person512