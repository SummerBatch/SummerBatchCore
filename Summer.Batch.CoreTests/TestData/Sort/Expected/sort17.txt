Person5   12
Person4   37
Person1   45
Person2   59
Person3   68
